### ISSUE TITLE
HTML5 attributes without value give fatal error in find-untranslated.

### ISSUE THREAD

**Issue by allcaps** [collective/i18ndude#15]
Seems to fail on html5 attributes that do not require a value. Eg: `multiple` and `checked`.

```
<input multiple accept="image/*;capture=camera" type="file" title="Attach a file" i18n:attributes="title">

<input checked type="checkbox" name="friendly_type_name:list" tal:attributes="value ctype">
```

The error is:

   src/ploneintranet/search/browser/templates/all_results.pt:31:39:
   -FATAL- - ERROR in document:
   <unknown>:31:39: not well-formed (invalid token)

**Comment by mauritsvanrees** [comment]
No idea currently.
But if anyone is interested in fixing this, the error is from a `xml.sax.SAXException` at https://github.com/collective/i18ndude/blob/3.4.2/src/i18ndude/script.py#L146

**Comment by mauritsvanrees** [comment]
Note that extracting (rebuild-pot) works fine.  find-untranslated has the error. So one angle of attack would be to let find-untranslated use the HTMLTALParser like extract.py does. But those are very different things, so this does not seem easy.

**Comment by ale-rt** [comment]
Probably this is fixed after #27

**Comment by mauritsvanrees** [comment]
No it's not. I wrote this comment after working on 27. :-) zope.tal actually has no problems with this. It's the find-untranslated command that has the problem. It uses xml.sax as parser, which falls over it. I wonder if lxml better at that, but it looks like find-untranslated would need a complete overhaul.

**Comment by mauritsvanrees** [comment]
I guess this was fixed at the end of 2015 in PR #32 or some other fix.
I get this error with i18ndude 3.4.0, but not with 4.0.1. (Latest is 5.0.1.)
So I close this.
